### DISCUSSION TITLE
What to do when I notice false identifications

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#513] [Q&A] [answered]
Hi,

I have started my Birdnet-Pi three days ago and mostly it runs very well and have fun listening and watching the results. However I also noticed a few false identifications. What to do with them? Can I report them?

Thank you.

**Comment by mcguirepr89** [comment, marked as answer]
https://github.com/mcguirepr89/BirdNET-Pi/discussions?discussions_q=false
